src/term/mod.rs

```diff
@@ -2053,6 +2056,80 @@ mod tests {
 
         assert_eq!(term.grid()[&cursor].c, '▒');
     }
+
+    #[test]
+    fn increase_font_size_works() {
+        let size = SizeInfo {
+            width: 21.0,
+            height: 51.0,
+            cell_width: 3.0,
+            cell_height: 3.0,
+            padding_x: 0.0,
+            padding_y: 0.0,
+        };
+        let config : Config = Default::default();
+        let mut term = Term::new(&config, size);
+        term.change_font_size(10.0);
+
+        let expected_font_size :Size = config.font().size() + Size::new(10.0);
+        assert_eq!(term.font_size, expected_font_size);
+    }
+
+    #[test]
+    fn decrease_font_size_works() {
+        let size = SizeInfo {
+            width: 21.0,
+            height: 51.0,
+            cell_width: 3.0,
+            cell_height: 3.0,
+            padding_x: 0.0,
+            padding_y: 0.0,
+        };
+        let config : Config = Default::default();
+        let mut term = Term::new(&config, size);
+        let negative_font_size_step :f32 = -10.0;
+        term.change_font_size(negative_font_size_step);
+
+        let expected_font_size :Size = config.font().size() + Size::new(negative_font_size_step);
+        assert_eq!(term.font_size, expected_font_size);
+    }
+
+    #[test]
+    fn attempt_do_decrease_font_below_threshold_defaults_to_font_size_step() {
```

Comment:
I think naming here is a bit too verboose. Something like `prevent_font_below_threshold_works()` might fall more in line with the other test names and is still short and precise? Even though I'd be in favor of dropping the `_works()` from the end of all tests, but since the others have this too, it should be fine.

Reply:
Done!